packages/babel-traverse/src/scope/index.js

```diff
@@ -615,6 +615,26 @@ export default class Scope {
       return this.isPure(node.value, constantsOnly);
     } else if (t.isUnaryExpression(node)) {
       return this.isPure(node.argument, constantsOnly);
+    } else if (t.isTaggedTemplateExpression(node)) {
+      if (t.isMemberExpression(node.tag)) {
+        const { object, property } = node.tag;
+        const { name } = object;
+        if (
+          t.isIdentifier(object) &&
```

Comment:
Thanks - I should have known there'd be a convenience method hiding in plain sight...